renderer/components/Tabs.tsx

```diff
@@ -14,15 +18,15 @@ const Tabs = ({ selectedTab, setSelectedTab }: TabsProps) => {
           setSelectedTab(0);
         }}
       >
-        Upscayl
+        {t("TITLE")}
```

Comment:
Since the Latin script is much longer than the Chinese script for "Settings", I'd recommend creating a translation for "Upscayl" so the two tabs have balanced widths. The best name I could come up with is "阿普升图" (phonetic translation of "Up" that sounds like a nickname + literally "raise image", which should be a valid phrase in Chinese), which hopefully satisfies the colloquial aesthetic.

Reply:
Again, probably we can take this with an individual issue.. I think for other languages too, there might be similar catchy things to write.. It definitely needs native speakers